lib/declarations.d.ts

```diff
@@ -384,11 +384,11 @@ interface IPathFilteringService {
 	isFileExcluded(file: string, rules: string[], rootDir: string): boolean
 }
 
-interface ICordovaMigrationService {
-	downloadCordovaMigrationData(): IFuture<void>;
+interface IFrameworkMigrationService {
+	downloadMigrationData(): IFuture<void>;
 	getSupportedVersions(): IFuture<string[]>;
-	pluginsForVersion(version: string): IFuture<string[]>;
-	migratePlugins(plugins: string[], fromVersion: string, toVersion: string): IFuture<string[]>;
+	pluginsForVersion?(version: string): IFuture<string[]>;
```

Comment:
In fact I'll extact another interface for CordovaMigration, that will extend this one. This way I'll keep the code cleaner